### DISCUSSION TITLE
Admin account

### DISCUSSION THREAD

**Discussion by kevimax** [documenso/documenso#1164] [General]
Hi !

How do you create an administrator account to manage a self-hotted instance

Thanks
:D

**Comment by Phoenix-Ignited-Tech** [comment]
I'm trying to figure this one out myself. I'll post if I figure it out.

**Reply by Phoenix-Ignited-Tech** [reply]
hi, @kevimax try admin@documenso.com as the email and a password of "password".

**Reply by kevimax** [reply]
hi @Phoenix-Ignited-Tech 
This doesn't work for me...strange

**Comment by Phoenix-Ignited-Tech** [comment]
It would only work if you seeded the database as here: https://github.com/documenso/documenso#local-development, however if the database was not seeded (such as when running in production I suppose), you would have to manually create an ADMIN user either using a db client such as DBeaver or psql via the command line. I have a tutorial here using psql in the command line: https://phoenixignited.tech/manually-create-a-documenso-admin-account-with-psql

**Comment by kevimax** [comment]
@Phoenix-Ignited-Tech you are a genius!
Thanks a lot
